gateway/src/apicast/configuration_loader/remote_v2.lua

```diff
@@ -341,23 +422,32 @@ function _M:services()
     return nil, 'no endpoint'
   end
 
-  local url = services_index_endpoint(endpoint)
-  local res, err = http_client.get(url)
-
-  if not res and err then
-    ngx.log(ngx.DEBUG, 'services get error: ', err, ' url: ', url)
-    return nil, err
-  end
+  local SERVICES_PER_PAGE = 500
```

Comment:
Is there any reason for this particular value? Do different values change performance in any reasonable way?

Reply:
That's the default value used in the 3scale API. The gateway still needs to fix the value because it needs to iterate comparing the number of returned results with that value. If the gateway does not fix that value, a change in the 3scale API can break the gateway pagination feature. 

In the future, we can expose as env var